### ISSUE TITLE
make regen in core fails

### ISSUE THREAD

**Issue by rurban** [perl11/p5-Config#4]
```
--- ext/Config/Config_xs.PL
+++ ext/Config/Config_xs.PL
@@ -23,6 +23,9 @@ $VERSION = '6.05';
 my $in_core;

 BEGIN {
+    if (-d 'ext/Config') {
+        chdir 'ext/Config';
+    }
```

**Comment by rurban** [comment]
See https://github.com/perl11/cperl/commit/6858c39da414d74069be7882f9209381a2189579 and
https://github.com/perl11/cperl/commit/47c2e1d15c544bfb185e05efe7b5a719f1a03127

this smoked fine in `smoke/mingw_cfg_fix`

**Comment by bulk88** [comment]
Fix published in  6.05_02.
